src/components/common/dialogs/CustomDialog.js

```diff
@@ -90,6 +90,8 @@ const CustomDialog = ({
       break
   }
 
+  const posthog = usePostHog()
```

Comment:
no need it's always will be undefined

```suggestion
```

Reply:
See below comment but this statement is not true. before login we have a custom dialog (securityDialog in case of an emergency and we need to block access)
for some reason there it does have a defined posthog

Reply:
Okay then let's create a custom provider performing this check:


```typescript
export const CustomPostHogProvider = ({ children }) => {
  if (!usePostHog()) {
    return (
      <PostHogProvider 
        apiKey={Config.posthogApiKey} 
        options={{ host: Config.posthogHost }} 
        autocapture={false}
      >
        {children}
      </PostHogProvider>
  }

  // already wrapped via provider
  return children
}
```

Reply:
right, that seems cleaner
fixed now